src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -20,6 +20,7 @@ import java.io.IOException
 import java.io.StringWriter
 import java.lang.reflect.Array
 import java.util.Arrays
+import java.util.Locale
```

Comment:
Makes sense. `KModifier` uses the same approach to printing, worth opening an issue to fix that as well?